server/changeWatcher.js

```diff
@@ -0,0 +1,14 @@
+const chokidar = require('chokidar');
+const file = require('./file');
```

Comment:
can do `const isCurrentFile = require('./file').isCurrentFile` if you only need 1 fn